index.js

```diff
@@ -220,8 +220,17 @@ async function fastifyUnderPressure (fastify, opts) {
     return false
   }
 
+  function getReqConfig(req) {
+    if (req.routeOptions && req.routeOptions.config) {
```

Comment:
```suggestion
    if (req.routeOptions?.config) {
```

Reply:
I don't think we need a function for this.

Reply:
`?.` is allowed? package.json doesn't have `engine` and code uses `||` for defaults

Reply:
Well, I started to contribute to Fastify for v5 so I am not very sure.
But for v4, [I think it's ok too](https://fastify.dev/docs/v4.28.x/Reference/LTS/#schedule).

Reply:
> ?. is allowed?

Yes

Reply:
Oh this is a backport, then no